vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -26,89 +27,90 @@ packages:
      - libvirt-python
      - lxml
     pip3:
-     - yq 
-     - virtualbmc 
-     - python-ironicclient 
-     - python-ironic-inspector-client 
-     - lxml 
-     - netaddr 
-     - requests 
-     - setuptools 
+     - yq
+     - virtualbmc
+     - python-ironicclient
+     - python-ironic-inspector-client
+     - lxml
+     - netaddr
+     - requests
+     - setuptools
      - libvirt-python
      - python-openstackclient
   centos:
     rhel7:
         packages:
-         - crudini 
-         - curl 
-         - dnsmasq 
-         - NetworkManager 
-         - nmap 
-         - patch 
-         - psmisc 
-         - python-pip 
-         - python-requests 
-         - python-setuptools 
-         - vim-enhanced 
+         - crudini
+         - curl
+         - dnsmasq
+         - NetworkManager
+         - nmap
+         - patch
```

Comment:
We're still missing the package in the centos7 case, I did yum install OVMF which appears to contain the right files

Reply:
It's a bit hard to follow since there are unrelated sort/linefeed changes here too.